SPECS/libvirt-glib/libvirt-glib.spec

```diff
@@ -0,0 +1,161 @@
+Summary:        libvirt glib integration for events
+Name:           libvirt-glib
+Version:        4.0.0
+Release:        4%{?dist}
+License:        LGPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://libvirt.org/
+Source0:        https://libvirt.org/sources/glib/%{name}-%{version}.tar.xz
+Patch1:         %{name}-%{version}-cast-align.patch
+
+BuildRequires:  gettext
+BuildRequires:  glib2-devel
+BuildRequires:  gobject-introspection-devel
+BuildRequires:  gtk-doc
+BuildRequires:  libvirt-devel
+BuildRequires:  libxml2-devel
+BuildRequires:  meson
+BuildRequires:  vala
+
+%description
+This package provides integration between libvirt and the glib
+event loop.
+
+%package devel
+Summary:        libvirt glib integration for events development files
+Requires:       %{name} = %{version}-%{release}
+
+%description devel
+This package provides development header files and libraries for
+integration between libvirt and the glib event loop.
+
+%package -n libvirt-gconfig
+Summary:        libvirt object APIs for processing object configuration
+
+%description -n libvirt-gconfig
+This package provides APIs for processing the object configuration
+data
+
+%package -n libvirt-gobject
+Summary:        libvirt object APIs for managing virtualization hosts
+
+%description -n libvirt-gobject
+This package provides APIs for managing virtualization host
+objects
+
+%package -n libvirt-gconfig-devel
+Summary:        libvirt object APIs for processing object configuration development files
+Requires:       libvirt-gconfig = %{version}-%{release}
+
+%description -n libvirt-gconfig-devel
+This package provides development header files and libraries for
+the object configuration APIs.
+
+%package -n libvirt-gobject-devel
+Summary:        libvirt object APIs for managing virtualization hosts development files
+Requires:       %{name}-devel = %{version}-%{release}
+Requires:       libvirt-devel
+Requires:       libvirt-gconfig-devel = %{version}-%{release}
+Requires:       libvirt-gobject = %{version}-%{release}
+
+%description -n libvirt-gobject-devel
+This package provides development header files and libraries for
+managing virtualization host objects
+
+%prep
+%autosetup -p1
+
+%build
+%meson -Drpath=disabled
+%meson_build
+
+%install
+%meson_install
+
+%find_lang %{name}
+
+%check
+%meson_test
+
+%ldconfig_scriptlets
+
+%ldconfig_scriptlets -n libvirt-gconfig
+
+%ldconfig_scriptlets -n libvirt-gobject
+
+%files -f %{name}.lang
+%license COPYING
+%doc README AUTHORS NEWS
+%{_libdir}/libvirt-glib-1.0.so.*
+%{_libdir}/girepository-1.0/LibvirtGLib-1.0.typelib
+
+%files -n libvirt-gconfig
+%{_libdir}/libvirt-gconfig-1.0.so.*
```

Comment:
We should package the license in `libvirt-gconfig` and `libvirt-gobject`, since they are installable independent of the main package.

Reply:
Good catch, thanks for noticing that!